### DISCUSSION TITLE
Add "copy codeblock" for non-prism highlighting

### DISCUSSION THREAD

**Discussion by qjcg** [colinwilson/lotusdocs#106] [Feature Requests]
With `params.docs.prism = true` set in `hugo.toml`, there's a "copy codeblock" button on code blocks that's very convenient.

It would be awesome if the same "copy codeblock" feature was available with `params.docs.prism = false` as well!

**Comment by colinwilson** [comment]
This is definitely a nice feature to have for those who prefer Chroma over PrismJS for syntax highlighting. Will implement this soon.

Thanks for the feature request.
